src/apps/reservation-list/modal/delete-reservation/delete-reservation-modal.tsx

```diff
@@ -74,16 +74,46 @@ const DeleteReservationModal: FC<DeleteReservationModalProps> = ({
   return (
     <Modal
       modalId={modalId}
-      classNames="modal-cta"
+      classNames="modal-cta modal-padding"
       closeModalAriaLabelText={t("deleteReservationModalCloseModalText")}
       screenReaderModalDescriptionText={t(
         "deleteReservationModalAriaDescriptionText"
       )}
     >
-      <DeleteReservationContent
-        deleteReservation={() => removeSelectedReservations()}
-        reservationsCount={reservations.length}
-      />
+      {["idle", "pending"].includes(requestStatus) && (
+        <DeleteReservationContent
+          deleteReservation={() => removeSelectedReservationsHandler()}
+          reservationsCount={reservations.length}
+          deletionStatus={requestStatus}
+        />
+      )}
+      {requestStatus === "success" && (
+        <ModalMessage
+          title={t("deleteReservationModalSuccessTitleText")}
+          subTitle={t("deleteReservationModalSuccessStatusText")}
+          ctaButton={{
+            text: t("deleteReservationModalButtonText"),
+            closeAllModals: true,
+            callback: () => {
+              setRequestStatus("idle");
+            }
+          }}
+        />
+      )}
+
+      {requestStatus === "error" && (
+        <ModalMessage
+          title={t("deleteReservationModalErrorsTitleText")}
+          subTitle={t("deleteReservationModalErrorsStatusText")}
+          ctaButton={{
+            text: t("deleteReservationModalButtonText"),
+            closeAllModals: true,
+            callback: () => {
+              setRequestStatus("idle");
+            }
+          }}
+        />
+      )}
```

Comment:
Seems a bit redundant...
Would prefer an if statement and ternary operator inside title and subTitle, but that is just me I guess

Reply:
I havde defined the ctaButton params once to remove redundant code. 👍